csclient/params/params.go

```diff
@@ -23,6 +23,10 @@ const (
 	// EntityIdHeader specifies the header attribute that will hold the
 	// id of the entity for archive GET responses.
 	EntityIdHeader = "Entity-Id"
+
+	// ResourceRevisionHeader specifies the header attribute that will hold the
+	// revision of the resource retrieved by a resource GET request.
+	ResourceRevisionHeader = "Resource-Revision"
```

Comment:
Ah, it's for the response.  Yuck.  So either it must be in a header, or we use multi-part response bodies, or we make revision required.